src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -366,13 +367,26 @@ public String serverVersion() {
 
     /**
      * Executes a HTTP request. This method provides a mechanism to extend the API
-     * <p><b>Note</b>: The response must be closed after use to release the connection.
+     * <p><b>Note</b>: Streams obtained from the HttpConnection must be closed after use to release
+     * the connection.
+     * </p>
+     * <pre>

```

Comment:
I'm not sure what the correct markup is here - do you need both `<pre>` and `{@code`?